python/ray/autoscaler/_private/command_runner.py

```diff
@@ -35,6 +35,8 @@
 HASH_MAX_LENGTH = 10
 KUBECTL_RSYNC = os.path.join(
     os.path.dirname(os.path.abspath(__file__)), "kubernetes/kubectl-rsync.sh")
+MAX_HOME_RETRIES = 3
+DELAY_BEFORE_HOME_RETRY = 5
```

Comment:
`HOME_RETRY_DELAY_S`

let's keep the numerical value present in the variable name?

Reply:
Yep -- don't forget your units, kids.